src/main/java/it/gov/pagopa/pu/migration/dto/debtpositiontypeorgoperator/DebtPositionTypeOrgOperatorMigrationFileResult.java

```diff
@@ -0,0 +1,20 @@
+package it.gov.pagopa.pu.migration.dto.debtpositiontypeorgoperator;
+
+import it.gov.pagopa.pu.migration.wf.dto.MigrationFileResult;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
+import java.util.List;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+@NoArgsConstructor
+@AllArgsConstructor
+@SuperBuilder
+@ToString(callSuper = true)
+public class DebtPositionTypeOrgOperatorMigrationFileResult extends MigrationFileResult {
+  private Long organizationId;
+  private Long numTotalRows;
+  private Long numCorrectlyProcessedRows;
+  private List<DebtPositionTypeOrgOperatorErrorDTO> errorList;
```

Comment:
why this List? what do you plan to do with this info on the DebtPositionTypeOrgOperatorMigrationFileResult?